aggregator/src/app/BundleService.ts

```diff
@@ -200,6 +201,11 @@ export default class BundleService {
     });
   }
 
+  async lookupBundle(hash: string) {
+    const bundleArray = await this.bundleTable.findBundle(hash);
+    return bundleArray[0] ? bundleArray[0] : nil;
```

Comment:
Why does `findBundle` return an array? Wouldn't `Bundle | nil` be more appropriate? That way you'd also not need to transform the result here.